From: Dimitri John Ledkov <xnox@ubuntu.com>
Date: Mon, 14 Sep 2020 10:51:32 +0100
Subject: configure.ac: one more dejavu font search path

Debian/Ubuntu ship dejavu font in a subdir of truetype.

Patch-Name: ubuntu-dejavu-font-path.patch
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 6a88b9b..fae9171 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1671,7 +1671,7 @@ fi
 
 if test x"$starfield_excuse" = x; then
    for ext in pcf pcf.gz bdf bdf.gz ttf ttf.gz; do
-     for dir in . /usr/src /usr/share/fonts/X11/misc /usr/share/fonts/truetype/ttf-dejavu /usr/share/fonts/dejavu /usr/share/fonts/truetype; do
+     for dir in . /usr/src /usr/share/fonts/X11/misc /usr/share/fonts/truetype/ttf-dejavu /usr/share/fonts/dejavu /usr/share/fonts/truetype /usr/share/fonts/truetype/dejavu; do
         if test -f "$dir/DejaVuSans.$ext"; then
           DJVU_FONT_SOURCE="$dir/DejaVuSans.$ext"
           break 2
